pages/edit-category/[id].js

```diff
@@ -0,0 +1,166 @@
+import Head from "next/head";
+import styled from "styled-components";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import { useCategoriesStore } from "../../useStore";
+
+// Components
+import Header from "../../components/Header";
+import { FormMain, StyledForm, Input } from "../../components/FormComponents";
+import { ButtonGroup, ButtonSmall } from "../../components/Button";
+import { IconChevronLeft, IconChevronRight } from "../../components/Icons";
+
+export default function EditCategory() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  //######################
+  //GET THINGS FROM STORE
+  //######################
```

Comment:
These big comments are very verbose. You might not need them, because you see in the next line, that you retrieving data from the store. ;-)

If you want them in the codebase, because for easier accessibility you can just get rid of the big '#`s